modules/tile-converter/src/3d-tiles-converter/helpers/tiles-converter.ts

```diff
@@ -0,0 +1,354 @@
+/* eslint-disable no-console */
+
+import type {I3STileContent, FeatureAttribute} from '@loaders.gl/i3s';
```

Comment:
Because this became a completely new file, it is not possible to easily see what has changed.